Translations/fr.po

```diff
@@ -743,8 +742,7 @@ msgstr "Non"
 
 #: Source/DiabloUI/support_lines.cpp:8
 msgid "GOG.com maintains a web site at https://www.gog.com/forum/diablo"
-msgstr ""
-"GOG.com maintient un site Web à l'adresse https://www.gog.com/forum/diablo"
+msgstr "GOG.com maintient un site à l'adresse https://www.gog.com/forum/diablo"
```

Comment:
False friend, I think "tient un site" would be better: https://www.linguee.fr/francais-anglais/search?source=auto&query=maintains+a+website

Reply:
I'm fine with "tient", may be "heberge" would be better?

Reply:
Ah yes, "heberge" would fit too!